pack.ps1

```diff
@@ -32,9 +34,7 @@ foreach ($project in $targetProjects)
     $projectPath = "$PSScriptRoot\src\$project\$project.csproj"
     $outputPath = "$PSScriptRoot\src\$project\$PublishRelativePath"
 
-    #
-    # The build system expects pre-built binaries to be in the folder pointed to by 'OutDir'.
-    dotnet pack -o "$outputPath" /p:OutDir=$PrebuiltBinariesDir "$projectPath" --no-build | Tee-Object -FilePath "$LogDirectory\build.log"
+    dotnet pack -c $BuildConfig -o "$outputPath" "$projectPath" --no-build | Tee-Object -FilePath "$LogDirectory\build.log"
```

Comment:
I'm a bit confused. Since we say `--no-build` here, why do we need a `-c $BuildConfig`?

Reply:
Oh, is that just to tell `dotnet pack` where to pick up binaries? Okay then.

Reply:
Yes, that's correct. The default configuration is `Debug` when using `--no-build`, so we explicitly override it. https://github.com/dotnet/sdk/issues/9570